conf/nginx/st2.conf

```diff
@@ -15,7 +15,9 @@ server {
   if ($ssl_protocol = "") {
        return 308 https://$host$request_uri;
   }
-
+  if ($request_method !~ ^(GET|HEAD|POST|PUT|DELETE)$ ) {
+       return 405;
+  }
```

Comment:
```suggestion
```
Similar to what @m4dcoder was saying before, looks like the directive added is not effective for the HTTP server section here as it's just redirecting to the https one.

Reply:
I have added a directive for HTTPS as well.
TRACE will be not be allowed in the HTTP method. Screenshots of testing for your reference.
![trace_nginx_request](https://user-images.githubusercontent.com/72917414/112125376-600c6d00-8be9-11eb-8213-732cd817b619.jpg)
![trace_nginx_response](https://user-images.githubusercontent.com/72917414/112125438-70bce300-8be9-11eb-8e22-93e05db0663d.jpg)

I have added max_ranges 0 to disable the partial content responses on Nginx. If a range is set as a byte then it is more vulnerable. It is recommended to not set the range as a byte.

Reply:
Thanks for the update.

As per my comment above - we also needs to support / whitelist ``OPTIONS`` that's used by st2web for CORS purposes.